packages/ui/.jest/a11y.test.tsx

```diff
@@ -1,140 +0,0 @@
-/**
- * TODO/ outdated script with vitest migration
- **/
```

Comment:
Why deleting it? 😢

Reply:
it's not used any more, and we be replace by the next pr of axe